.github/workflows/ci.yml

```diff
@@ -79,7 +79,8 @@ jobs:
 
   # Build github-metrics-aggregator image and push to artifact registry
   build:
-    runs-on: 'ubuntu-latest'
+    runs-on:
```

Comment:
Can you add a comment here with some background context?

Reply:
I figured I wouldn't actually merge this, as pdewilde-test is probably not the final label we will end up with

But unfortunatly it doesn't seem to be working at all